game/src/sandbox/dashboards/parking_overhead.rs

```diff
@@ -1,20 +1,104 @@
 use geom::Duration;
 use sim::{TripEndpoint, TripID, TripPhaseType};
 use widgetry::table::{Col, Filter, Table};
-use widgetry::{EventCtx, Filler, Line, Panel, State, Text, Toggle, Widget};
+use widgetry::{EventCtx, Filler, GfxCtx, Line, Outcome, Panel, State, Text, Toggle, Widget};
+
+type Transition = widgetry::Transition<App>;
 
 use crate::app::App;
-use crate::sandbox::dashboards::generic_trip_table::GenericTripTable;
+use crate::sandbox::dashboards::generic_trip_table::{open_trip_transition, preview_trip};
 use crate::sandbox::dashboards::DashTab;
 
 // TODO Compare all of these things before/after
 
-pub struct ParkingOverhead;
+pub struct ParkingOverhead {
+    tab: DashTab,
+    table: Table<App, Entry, Filters>,
+    panel: Panel,
+}
 
 impl ParkingOverhead {
     pub fn new(ctx: &mut EventCtx, app: &App) -> Box<dyn State<App>> {
         let table = make_table(app);
-        GenericTripTable::new(ctx, app, DashTab::ParkingOverhead, table, make_panel)
+        let col = Widget::col(vec![
+            DashTab::ParkingOverhead.picker(ctx, app),
+            Widget::col(vec![
+                Widget::row(vec![
+                    Text::from_multiline(vec![
```

Comment:
No changes to the content, other than that's it's now outlined and has "section" styling.